source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -0,0 +1,229 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Ways of Working
+last_reviewed_on: 2024-07-18
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## What are our Core Values?
+
+- **Collaboration:** Foster a culture of open communication and collective problem-solving.
+- **Quality:** Uphold high standards in our work, ensuring reliability and efficiency.
+- **Innovation:** Embrace new ideas and technologies that propel us and our platforms forward.
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress.
+
+## Team Roles
+
+- **Team Members:** Comprising a mix of developers, data, platform and software engineers, responsible for the delivery of tasks and active participation in all Agile ceremonies.
+- **Product Manager (PM):** Oversees the product backlog, ensuring it aligns with stakeholder needs and business objectives.
+- **Delivery Manager (DM):** Facilitates the Agile process, removing blockers and ensuring timelines are met.
```

Comment:
Technical Architect?

Reply:
I don't do anything apparently

Reply:
I am not falling into such an obvious TARP.